internal/pkg/flink/internal/controller/input_controller_test.go

```diff
@@ -204,7 +204,7 @@ func (s *InputControllerTestSuite) TestRenderMsgAndStatusLocalStatements() {
 		{
 			name:      "local failed statement",
 			statement: &types.ProcessedStatement{IsLocalStatement: true, Status: types.FAILED},
-			want:      "Error: Couldn't process statement. Please check your statement and try again\n",
+			want:      "Error: Couldn't process statement. Please check your statement and try again.\n",
```

Comment:
Errors in Go should not be capitalized or end with punctuation
```suggestion
			want:      "Error: couldn't process statement, please check your statement and try again\n",
```

Reply:
The second half of this error seems like more of a suggestion than an error. I believe @YannickPferr wrote some code to separate those.

Reply:
> Errors in Go should not be capitalized or end with punctuation

Yeah, that's the default behavior. But I think there are multiple reasons for us to format it in the regular human readable way: 

- We just display errors sent from the backend and they format it properly there so we would have inconsistent formatting for local/remote errors:

<img width="536" alt="image" src="https://github.com/confluentinc/cli/assets/11739405/cc5f9126-0d37-42a4-b43b-9e6a186f1eb0">


- Also, we're building the UI to end users and I think it makes sense to format it in a more readable way. It's not like a log file that's going to be read sometimes from a dev. If only wrote local errors in lowercase, we would have `error: couldn't process statement, please check your statement and try again` and info and warnings right next to it properly capitalized `Statement successfully submitted`, which wouldn't also really look consistent..

Reply:
I've reverted, so the errors are now lowercase so we merge this for now. But I still think it's makes sense to format this properly but we can do that later :)

>The second half of this error seems like more of a suggestion than an error. I believe @YannickPferr wrote some code to separate those.

I've used the logic Yannick added and adjusted the messages. Think having` "Suggestion: check your statement and try again"` isn't really helpful so I kept that inside the error.